lib/src/DatePicker/DatePickerToolbar.tsx

```diff
@@ -26,50 +23,32 @@ export const useStyles = makeStyles(
 export const DatePickerToolbar: React.FC<ToolbarComponentProps> = ({
   date,
   views,
-  setOpenView,
   isLandscape,
-  openView,
+  title = 'SELECT DATE',
```

Comment:
This would make a good new addition to https://material-ui.com/guides/localization/.
What about using lower case, so that people can more easily tweak the capitalization with CSS. For instance, they might have globally disabled the uppercase of the button.

```suggestion
  title = 'Select date',
```